tests/integ/test_udtf.py

```diff
@@ -284,18 +284,18 @@ def process(
 
 @pytest.mark.skipif(not is_pandas_available, reason="pandas is required")
 def test_apply_in_pandas(session):
-    # test with element wise opeartion
+    # test with element wise operation
     def convert(pdf):
-        pdf.columns = ["location", "temp_c"]
         return pdf.assign(temp_f=lambda x: x.temp_c * 9 / 5 + 32)
 
     df = session.createDataFrame(
         [("SF", 21.0), ("SF", 17.5), ("SF", 24.0), ("NY", 30.9), ("NY", 33.6)],
-        schema=["location", "temp_c"],
+        schema=['"location"', '"temp_c"'],
     )
 
-    df = df.group_by("location").apply_in_pandas(
+    df = df.group_by('"location"').apply_in_pandas(
```

Comment:
does the old code not work with this change? That could be a BCR if so

Reply:
ah thanks for catching, these are not needed, removing...